support/@trustfractal/data-host/src/lib/fractalUI.ts

```diff
@@ -0,0 +1,11 @@
+// import App from '@components/App';
+import ReactDOM from 'react-dom';
+
+import App from '../popup/components/App';
+
+export class FractalUI {
+  // substrateAddress is going to be used to send funds
+  async render(element: HTMLElement | null): Promise<void> {
```

Comment:
What is the point of this returning a promise? I know we'd want to do async stuff, but would users waiting for the promise to resolve mean anything to them?